store/iavl/store.go

```diff
@@ -273,7 +273,13 @@ func (st *Store) Export(version int64) (*iavl.Exporter, error) {
 	if !ok || tree == nil {
 		return nil, fmt.Errorf("iavl export failed: unable to fetch tree for version %v", version)
 	}
-	return tree.Export(), nil
+	exporter := tree.Export()
```

Comment:
```suggestion

	exporter := tree.Export()
```

Reply:
This change is no longer necessary for the scope of this PR branch